easybuild/easyconfigs/x/X11/X11-20170129-GCCcore-6.3.0.eb

```diff
@@ -35,90 +35,241 @@ builddependencies = [
+    }),
+    ('randrproto', '1.5.0', {  # 2015-05-17
+        'checksums': ['8f8a716d6daa6ba05df97d513960d35a39e040600bf04b313633f11679006fab'],
+    }),
+    ('recordproto', '1.14.2', {  # 2012-03-23
+        'checksums': ['485f792570dd7afe49144227f325bf2827bc7d87aae6a8ab6c1de2b06b1c68c5'],
+    }),
+    ('renderproto', '0.11.1', {  # 2010-08-10
+        'checksums': ['a0a4be3cad9381ae28279ba5582e679491fc2bec9aab8a65993108bf8dbce5fe'],
+    }),
+    ('resourceproto', '1.2.0', {  # 2011-05-28
+        'checksums': ['469029d14fdeeaa7eed1be585998ff4cb92cf664f872d1d69c04140815b78734'],
+    }),
+    ('scrnsaverproto', '1.2.2', {  # 2012-03-23
+        'checksums': ['d8dee19c52977f65af08fad6aa237bacee11bc5a33e1b9b064e8ac1fd99d6e79'],
+    }),
+    ('videoproto', '2.3.3', {  # 2016-03-11
+        'checksums': ['df8dfeb158767f843054248d020e291a2c40f7f5e0ac6d8706966686fee7c5c0'],
+    }),
+    ('xcmiscproto', '1.2.2', {  # 2012-03-23
+        'checksums': ['48013cfbe4bd5580925a854a43e2bccbb4c7a5a31128070644617b6dc7f8ef85'],
+    }),
+    ('xextproto', '7.3.0', {  # 2013-12-27
+        'checksums': ['1b1bcdf91221e78c6c33738667a57bd9aaa63d5953174ad8ed9929296741c9f5'],
+    }),
+    ('xf86bigfontproto', '1.2.0', {  # 2009-08-27
+        'checksums': ['d190e6462b2bbbac6ee9a007fb8eccb9ad9f5f70544154f388266f031d4bbb23'],
+    }),
+    ('xf86dgaproto', '2.1', {  # 2009-10-01
+        'checksums': ['73bc6fc830cce5a0ec9c750d4702601fc0fca12d6353ede8b4c0092c9c4ca2af'],
+    }),
+    ('xf86driproto', '2.1.1', {  # 2011-01-06
+        'checksums': ['18ff8de129b89fa24a412a1ec1799f8687f96c186c655b44b1a714a3a5d15d6c'],
+    }),
+    ('xf86vidmodeproto', '2.3.1', {  # 2011-01-06
+        'checksums': ['c3512b11cefa7558576551f8582c6e7071c8a24d78176059d94b84b48b262979'],
+    }),
+    ('xineramaproto', '1.2.1', {  # 2011-01-06
+        'checksums': ['d99e121edf7b310008d7371ac5dbe3aa2810996d476b754dc78477cc26e5e7c1'],
+    }),
+    ('xproto', '7.0.31', {  # 2016-09-23
+        'checksums': ['6d755eaae27b45c5cc75529a12855fed5de5969b367ed05003944cf901ed43c7'],
+    }),
+    ('libXau', '1.0.8', {  # 2013-05-24
+        'checksums': ['c343b4ef66d66a6b3e0e27aa46b37ad5cab0f11a5c565eafb4a1c7590bc71d7b'],
+    }),
+    ('libXdmcp', '1.1.2', {  # 2015-03-21
+        'checksums': ['6f7c7e491a23035a26284d247779174dedc67e34e93cc3548b648ffdb6fc57c0'],
+    }),
+    ('xcb-proto', '1.12', {  # 2016-05-18
+        'checksums': ['cfa49e65dd390233d560ce4476575e4b76e505a0e0bacdfb5ba6f8d0af53fd59'],
+    }),
+    ('libxcb', '1.12', {  # 2016-05-18
+        'sources': [SOURCE_TAR_GZ],
+        'patches': ['libxcb-%(version)s_indent.patch'],
+        'checksums': [
+            '092f147149d8a6410647a848378aaae749304d5b73e028ccb8306aa8a9e26f06',  # libxcb-1.12.tar.gz
+            'bf6fb7f37b756fad5b6e1b93c3c8bb3ea60371f2b71b08249ee5a4d4a12b8df5',  # libxcb-1.12_indent.patch
+        ],
+        'start_dir': '%(name)s-%(version)s',
+
+    }),
```

Comment:
Why the blankline? (Didn't notice that on the 2016xx ones)

Reply:
It hitchhiked along from the `X11` 2018* bundle where this was copied from, fixed in 17ad7218b